javalib/src/main/scala/java/util/concurrent/atomic/AtomicReference.scala

```diff
@@ -1,5 +1,14 @@
 package java.util.concurrent.atomic
 
+// 2019-04-25 Note Well!
```

Comment:
I don't think the dates serve any purpose in the source code. You can always git blame if you want to find when it was added. Please drop this line.

Reply:
Changed as requested. Travis all green